modules/sc-mesh-secure-deployment/src/nats/cbma/secure_socket/server.py

```diff
@@ -32,7 +32,7 @@ def __init__(self,
         self.client_ipv6 = client_ipv6
         self.macsec_callback = macsec_callback
 
-        super().__init__(certificates, SSL.TLSv1_2_METHOD)
+        super().__init__(certificates, SSL.SSLv23_METHOD)
```

Comment:
I think this will work, although it has deprecation warning.

The newer way is to pass SSL.TLS_SERVER_METHOD and set minimum version by ctx options.

Reply:
Same as https://github.com/tiiuae/mesh_com/pull/465#discussion_r1661874328